selftests/unit/test_test.py

```diff
@@ -1,11 +1,7 @@
 import os
 import shutil
 import tempfile
-import unittest
-try:
-    from unittest import mock
-except ImportError:
-    import mock
+import unittest.mock
```

Comment:
Is there a reason for...
import unittest.mock
... instead of...
from unittest import mock
?

I see both patterns on those changes. Maybe we should stick with a single pattern?

Reply:
Not really... I agree, one pattern is definitely better.